dotCMS/src/test/java/com/dotmarketing/business/cache/provider/h22/H22CacheTest.java

```diff
@@ -32,65 +34,88 @@
 					+ "C"
 					+ "ASDSADFDDSFasfddsadasdsadsadasdq3r4efwqrrqwerqewrqewrqreqwreqwrqewrqwerqewrqewrqwerqwerqwerqewr43545324542354235243524354325423qwerewds fds fds gf eqgq ewg qeg qe wg egw	ww	eeR ASdsadsadadsadsadsadaqrewq43223t14@#$#@^%$%&%#$sfwf	erqwfewfqewfgqewdsfqewtr243fq43f4q444fa4ferfrearge");;
 	final String CANT_CACHE_KEYNAME = "CantCacheMe";
-	final int numberOfPuts = 5000;
+	final int numberOfPuts = 1000;
 	final int numberOfThreads = 40;
 	final int numberOfGroups = 100;
 	final int maxCharOfObjects = 100;
+	static H22Cache cache ;
+	
+	@BeforeClass
+	public static void initCache(){
+	       // File dir = Files.createTempDir();
+        File dir = new File("/tmp/h2cachetest");
+        FileUtil.deltree(dir);
+        dir.mkdirs();
+
+        cache = new H22Cache(dir.getAbsolutePath());
+        try {
+            cache.init();
+        }
+        catch(Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5918493)